src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -142,26 +142,37 @@
 ;;; Public
 
 (schema/defn ^:always-validate
+  initialize-lifecycle-fns :- jruby-schemas/LifecycleFns
+  [config :- (schema/maybe {schema/Keyword schema/Any})]
```

Comment:
This can have a more strict schema than this, right?  We know what some of the required keys in there are?